tests/python/accuracy/public_scope.json

```diff
@@ -52,14 +52,31 @@
                     "(50, 278, 364, 382, 0.913, 18, sheep)"
                 ]
             }
-        ]
+        ],
+        "TODO: there is a bug in openvino": {
```

Comment:
What bug? Any reference to the ticket in the code would be helpful.

Reply:
Added